sessions/js-structure/quiz-app/global.css

```diff
@@ -0,0 +1,66 @@
+:root {
+	--primary: #ffffff;
+	--secondary: #ff4a11;
+	--secondary-light: #fddcd2;
+	--white: #ffffff;
+	--black: #252629;
+	--ff-primary: sans-serif;
+	--ff-serif: serif;
+	--fs-200: 0.6rem;
+	--fs-300: 0.8rem;
+	--fs-400: 1rem;
+	--fs-500: 1.1rem;
+	--fs-600: 1.5rem;
+	--fs-900: 1.75rem;
```

Comment:
I'm a little unsure about this. Usually these increments (100-based) are reserved for font-weight, rather than font-size.

Reply:
The naming is taken from Alex, as far as I remember. But I agree that the increments are slightly unusual. I adapted them to the tailwind naming (xs, sm, base, lg, 2xl, 3xl), what do you think?

Reply:
As per our conversation: let's just get rid of it ;)